traitlets/traitlets.py

```diff
@@ -649,14 +697,23 @@ def observe(*names):
     """
     return ObserveHandler(names)
 
+def validate(name):
+    """ A decorator which can be used to cross validate a member on a class.
 
-class ObserveHandler(BaseDescriptor):
+    Parameters
+    ----------
+    name
+        The str name of the attribute to observe on the object.
+    """
+    try:
+        len(name)
+        raise TraitError("Only one cross-validator is allowed per trait")
```

Comment:
I'm not sure of the logic here.  If the name has a length, then don't allow it.  For any error, then return the validator?